vaccine_feed_ingest/runners/ca/metrolink/parse.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import pathlib
+import sys
+
+from bs4 import BeautifulSoup
+
+input_dir = pathlib.Path(sys.argv[2])
+output_dir = pathlib.Path(sys.argv[1])
+
+input_filenames = [p for p in pathlib.Path(input_dir).iterdir() if p.is_file()]
+
+for filename in input_filenames:
+    sites = []
+
+    with filename.open() as fin:
+        content = fin.read()
+
+    soup = BeautifulSoup(content, "html.parser")
+    table = soup.find(id="vaxLocationsTable")
+    for row in table.find("tbody").find_all("tr"):
+        cells = row.find_all("td")
+
+        site = {
+            "site_name": str(cells[0].renderContents())[2:-1].split(" <br/> ")[0],
+            "site_address": str(cells[0].renderContents())[2:-1].split(" <br/> ")[1],
```

Comment:
Consider pulling the city out?  Matching against "(.*) <br/> (.*), (.*)" and taking group 3 should work.

Reply:
Sure; here's a sample from commit bdfefbe:

```json
{"name": "Pomona Fairplex (Gate 15)", "address": "2352 Arrow Hwy.", "city": "Pomona", "metrolink_line": "San Bernardino", "metrolink_station": "Pomona - North"}
{"name": "Cal State Northridge", "address": "18343 Plummer Street", "city": "Northridge", "metrolink_line": "Ventura County", "metrolink_station": "Northridge"}
{"name": "El Sereno Recreation Center", "address": "4721 Klamath St.", "city": "Los Angeles", "metrolink_line": "San Bernardino", "metrolink_station": "Cal State L.A."}
{"name": "St. John's Well Child and Family-Lincoln", "address": "2515 Alta St.", "city": "Los Angeles", "metrolink_line": "Union Station", "metrolink_station": "Union Station"}
```